src/modules/callouts/pages/CalloutAdminOverviewPage.vue

```diff
@@ -110,15 +110,29 @@ import {
 import ActionButton from '../components/ActionButton.vue';
 import CalloutSummary from '../../../components/CalloutSummary.vue';
 import AppAlert from '../../../components/AppAlert.vue';
+import { updateCallout, createCallout } from '../../../utils/api/callout';
 
 const props = defineProps<{
   callout: GetMoreCalloutData;
 }>();
 const { t } = useI18n();
 
 const route = useRoute();
+const router = useRouter();
 const wasJustCreated = route.query.created !== undefined;
 const wasJustUpdated = route.query.updated !== undefined;
 
 const calloutLink = computed(() => `/callouts/${props.callout.slug}`);
+
+async function replicateThisCallout() {
+  var newCalloutData = { ...props.callout };
+  newCalloutData.slug = newCalloutData.slug + '-copy';
+  newCalloutData.title = newCalloutData.title + ' copy';
+  delete newCalloutData['status'];
+  const newCallout = await createCallout(newCalloutData);
+  router.push({
+    path: '/admin/callouts/view/' + newCallout.slug,
+    query: { ['replicated']: null },
```

Comment:
`replicated` doesn't need square brackets or quote marks, square brackets are only useful if you want do define a key based on a variable, i.e.
```ts
{replicated: null} === {['replicated']: null}
```

Reply:
ah! I had added that while bug hunting and it just stayed, thanks for noticing :)